pkgs/development/python-modules/scikit-survival/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchgit
+, git
+, setuptools_scm
+, cython
+, scikitlearn
+, cvxpy
+, cvxopt
+, numexpr
+, osqp
+, pandas
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-survival";
+  version = "0.11";
+
+  # temporary : wait for PR to be merged to go back on official
+  # motivation : requirements.txt was locking version of pandas and osqp
+  src = fetchgit {
+    url = "https://github.com/GuillaumeDesforges/scikit-survival";
+    rev = "a181b739500ba3a6c8553052dfefaca42ac501cd";
+    deepClone = true;  # submodules required
+    leaveDotGit = true;  # required for setuptools_cm
+
+    # warning : might not be deterministic, see https://github.com/NixOS/nixpkgs/issues/8567
+    sha256 = "0nf8ghs6xgqrxbsvy0vgm0wdw7y6x69zvac7hh9rh3gqygiyfkqc";
+  };
```

Comment:
this likely isn't needed if you do:
```
  postPatch = ''
    substituteInPlace setup.py \
      --replace "use_scm_version=True," 'version="${version}",'
  '';

Reply:
Hell, that was what I was looking for! Thanks